plex_trakt_sync/commands/watch.py

```diff
@@ -30,8 +32,35 @@ def __call__(self, message):
             print("%r: %.6F%% Watched: %s, LastViewed: %s" % (
                 movie, percent, movie.isWatched, movie.lastViewedAt
             ))
+            
+            
+            if factory.config()['watch']['username_filter']:
+                plex_sessions = self.plex.get_sessions()
+                current_sessionKeys = []
+                
+                for session in plex_sessions:
+                    current_sessionKeys.append(session.sessionKey)
+                
+                if 1 in current_sessionKeys and not self.sessions_cleared:
```

Comment:
please explain what is this magic `1`

Reply:
If there is a playing session when the Plex server restarts, it's sessionKey resets to 1 when the server comes back up. I can't find a way to be notified when the server restarts, so I figure if there is a sessionKey == 1, it means that the server has restarted so the sessions_from_username list has to be cleared and updated with the new sessionKey.

Reply:
please post those responses, as asked already:
- https://github.com/Taxel/PlexTraktSync/pull/480#issuecomment-916071825

Reply:
I do not understand what you mean by responses. Do you mean the console output?

```
Found <tmdb:170258:<Episode:326476:The-Office-s00e09>>
<Episode:326476:The-Office-s00e09>: 0.000000% Watched: False, LastViewed: None
New session from imjuzcy: 36
From username's session 36
State: playing
Found <tmdb:170258:<Episode:326476:The-Office-s00e09>>
<Episode:326476:The-Office-s00e09>: 2.121141% Watched: False, LastViewed: None
From username's session 36
State: paused
Found <tmdb:170258:<Episode:326476:The-Office-s00e09>>
<Episode:326476:The-Office-s00e09>: 2.356823% Watched: False, LastViewed: None
From username's session 36
State: stopped
Found <tmdb:170258:<Episode:326476:The-Office-s00e09>>
<Episode:326476:The-Office-s00e09>: 2.356823% Watched: False, LastViewed: None
From username's session 36
Remaining sessions from imjuzcy: []
```

Do you think it's too much output?

Reply:
I mean those api responses. `print(self.plex.get_sessions())`

Reply:
`print(self.plex.get_sessions())` :
```
[<Episode:192182:The-Office-s02e04>]
```

This `Episode` object is different than the `movie` object in the function.
`print(self.plex.get_sessions()[0].usernames)` : 
```
['imjuzcy']
```

`print(movie.usernames)` :
```
[]
```